core/src/main/java/tachyon/client/RemoteBlockInStream.java

```diff
@@ -126,6 +174,7 @@
     }
 
     mBlockInfo = mFile.getClientBlockInfo(mBlockIndex);
+    mSortedWorkers = sortLocations(mBlockInfo);
```

Comment:
Whoops I was calling the version of readRemoteByteBuffer that builds the sorted workers every time. I'm changing it to call the version that takes in a list of sorted workers.